optimize_versions.json

```diff
@@ -1 +1 @@
-["3.10.0", "3.9.0", "3.8.0", "3.7.0"]
```

Comment:
Associated with this change would be the removal of all optimize 3.7.0 docs from this branch.